src/cpp/redis.cpp

```diff
@@ -356,15 +361,24 @@ inline void Redis::_connect(std::string address_port)
             if(this->_redis->ping().compare("PONG")==0) {
                 break;
             }
+            else {
+                run_sleep = true;
+            }
         }
         catch (std::exception& e) {
             if(i == n_trials) {
-                delete this->_redis;
-                this->_redis = 0;
```

Comment:
Leave in the assignment to NULL (and change from 0 to NULL)

Reply:
I see what you are saying, but I think we should put the assignment to NULL up in teh construction portion, not the PONG portion

Reply:
Agree, keep assignment with the deletion